cmd/backup-manager/app/util/util.go

```diff
@@ -182,10 +178,10 @@ func ConstructBRGlobalOptionsForBackup(backup *v1alpha1.Backup) ([]string, error
 // ConstructMydumperOptionsForBackup constructs mydumper options for backup
 func ConstructMydumperOptionsForBackup(backup *v1alpha1.Backup) []string {
```

Comment:
update the function name too

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2870/commits/47d6d41707c75f880b1e31f410574272cf4ed3a4